src/BenchmarksWorkers/Workers/SignalRSerializer.cs

```diff
@@ -0,0 +1,289 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+using Benchmarks.ClientJob;
+using Benchmarks.ServerJob;
+using Newtonsoft.Json;
+
+namespace BenchmarksWorkers.Workers
+{
+    class SignalRSerializer : IResultsSerializer
+    {
+        public async Task InitializeDatabaseAsync(string connectionString, string tableName)
+        {
+            string createCmd =
+                @"
+                IF OBJECT_ID(N'dbo." + tableName + @"', N'U') IS NULL
+                BEGIN
+                    CREATE TABLE [dbo].[" + tableName + @"](
+                        [Id] [int] IDENTITY(1,1) NOT NULL PRIMARY KEY,
+                        [Excluded] [bit] DEFAULT 0,
+                        [DateTime] [datetimeoffset](7) NOT NULL,
+                        [Session] [nvarchar](200) NOT NULL,
+                        [Description] [nvarchar](200),
+                        [AspNetCoreVersion] [nvarchar](50) NOT NULL,
+                        [RuntimeVersion] [nvarchar](50) NOT NULL,
+                        [Scenario] [nvarchar](50) NOT NULL,
+                        [Hardware] [nvarchar](50) NOT NULL,
+                        [HardwareVersion] [nvarchar](128) NOT NULL,
+                        [OperatingSystem] [nvarchar](50) NOT NULL,
+                        [Framework] [nvarchar](50) NOT NULL,
+                        [RuntimeStore] [bit] NOT NULL,
+                        [Scheme] [nvarchar](50) NOT NULL,
+                        [Sources] [nvarchar](50) NULL,
+                        [ConnectionFilter] [nvarchar](50) NULL,
```

Comment:
Do you need that?

Reply:
Removed a couple items, ConnectionFilter and Method